xstate-chatbot/nodejs/src/machine/service/gis-service.js

```diff
@@ -138,6 +149,30 @@ class GisService {
       console.error(`API responded with ${JSON.stringify(responseBody)}`);
     }
   }
+  
+  async getFileForFileStoreId(filestoreId) {
+    let url = config.egovServices.egovServicesHost + config.egovServices.egovFilestoreServiceDownloadEndpoint;
+    url = `${url}?`;
+    url = `${url}tenantId=${config.rootTenantId}`;
+    url = `${url}&`;
+    url = `${url}fileStoreIds=${filestoreId}`;
+
+    const options = {
+      method: 'GET',
+      origin: '*',
+    };
+
+    let response = await fetch(url, options);
+    response = await (response).json();
+    const fileURL = response.fileStoreIds[0].url.split(',');
+    let fileName = geturl.parse(fileURL[0]);
+    fileName = path.basename(fileName.pathname);
+    fileName = fileName.substring(13);
+    await this.downloadImage(fileURL[0].toString(), fileName);
+    const file = fs.readFileSync(fileName, 'base64');
```

Comment:
From where you are calling this function you will get this file object. 

You need to import 'fs' like [here](https://github.com/egovernments/core-services/blob/d43c421c9330c1c9937fc3bd0dcf43de3f1cc59f/xstate-chatbot/nodejs/src/channel/value-first.js#L4)

Reply:
added and committed